server/routes/index.js

```diff
@@ -15,5 +16,6 @@ route.use('/auth', auth);
 route.use('/profiles', profile, follower);
 route.use('/user', userFollower);
 route.use('/articles', article);
+route.use('/articles', comment);
```

Comment:
You can add the comment beside the article instead of repeating route.use ('/articles)